.integration/tests/dpc_testnet1.rs

```diff
@@ -276,7 +275,7 @@ fn test_testnet1_dpc_execute_constraints() {
     println!("=========================================================");
     let num_constraints = inner_circuit_cs.num_constraints();
     println!("Inner circuit num constraints: {:?}", num_constraints);
-    assert_eq!(287247, num_constraints);
+    assert_eq!(321107, num_constraints);
```

Comment:
This increase is due to the underpaying of record commitment and program ID commitment trees.